src/transformers/modeling_bert.py

```diff
@@ -956,7 +956,7 @@ def __init__(self, config):
         super().__init__(config)
 
         if not config.is_decoder:
-            logger.info("If you want to use `BertLMHeadModel` as a standalone, add `is_decoder=True.`")
+            logger.warning("If you want to use `BertLMHeadModel` as a standalone, add `is_decoder=True.`")
```

Comment:
@LysandreJik - I changed it to a warning becasue it's actually wrong IMO to use `BertLMHeadModel` with bi-directional self-attention

Reply:
You're right, it's better.